### PR TITLE
As an appeal manager, I should be able to create/view/edit/disable/delete appeals

### PR DESCRIPTION
**Pull request by duozersk**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -74,6 +74,9 @@
     "cweagans/composer-patches": "^1.6.0",
     "drupal-composer/drupal-scaffold": "^2.5.0",
     "drupal/admin_toolbar": "^1.23",
+    "drupal/commerce": "^2.9",
+    "drupal/commerce_stripe": "^1.0@beta",
```
Comment (kalabro) [inline composer.json]: I'd remove payment methods for now. We will be using Realex for Credit Cards payments. For PayPal, we will need to investigate what's inside of `commerce_paypal` before making the decision.

File: modules/custom/falcon_development/config/install/features.bundle.falcon_feature.yml [hunk 2]
```diff
@@ -11,7 +11,7 @@ assignments:
   alter:
     core: true
     uuid: true
-    user_permissions: true
+    user_permissions: false
```
Comment (kalabro) [inline modules/custom/falcon_development/config/install/features.bundle.falcon_feature.yml:14]: Out of curiosity, why did we disable it?

File: modules/custom/falcon_commerce/falcon_commerce.install [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    'administrative_area' => 'DE',
+    'postal_code' => '19901',
+  ];
+
+  // The currency code.
+  $currency = 'EUR';
+
+  // Import the currency.
+  $currency_importer = \Drupal::service('commerce_price.currency_importer');
+  $currency_importer->import($currency);
```
Comment (kalabro) [inline modules/custom/falcon_commerce/falcon_commerce.install]: Why do we need to import the currency if there is a config `commerce_price.commerce_currency.EUR.yml` for it?

File: modules/custom/falcon_commerce/falcon_commerce.install [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    'country_code' => 'US',
+    'address_line1' => '8 The Green',
+    'address_line2' => 'Ste A',
+    'locality' => 'Dover',
+    'administrative_area' => 'DE',
+    'postal_code' => '19901',
+  ];
+
+  // The currency code.
+  $currency = 'EUR';
```
Comment (kalabro) [inline modules/custom/falcon_commerce/falcon_commerce.install]: I'm guessing that the default currency should be configurable during Falcon installation. Do you have any ideas how we can achieve it?

File: modules/features/falcon_feature_commerce/falcon_feature_commerce.info.yml [hunk 5]
```diff
@@ -0,0 +1,29 @@
+name: 'Falcon Commerce'
+type: module
+core: 8.x
+dependencies:
```
Comment (kalabro) [inline modules/features/falcon_feature_commerce/falcon_feature_commerce.info.yml]: Please remove payment methods for now

File: modules/features/falcon_feature_user/config/install/user.role.anonymous.yml [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+status: true
+dependencies: {  }
+id: anonymous
+label: 'Anonymous user'
+weight: 0
+is_admin: false
+permissions:
+  - 'access checkout'
+  - 'access content'
+  - 'view commerce_product'
```
Comment (kalabro) [inline modules/features/falcon_feature_user/config/install/user.role.anonymous.yml]: Do we need anonymous and authenticated roles to have access to products in decoupled architecture?

File: modules/features/falcon_feature_user/config/install/user.role.authenticated.yml [hunk 7]
```diff
@@ -0,0 +1,14 @@
+langcode: en
+status: true
+dependencies: {  }
+id: authenticated
+label: 'Authenticated user'
+weight: 1
+is_admin: false
+permissions:
```
Comment (kalabro) [inline modules/features/falcon_feature_user/config/install/user.role.authenticated.yml:8]: Neither of given commerce permissions seem to be required

File: modules/features/falcon_feature_user/config/install/user.role.donations_manager.yml [hunk 8]
```diff
@@ -0,0 +1,10 @@
+langcode: en
+status: true
+dependencies: {  }
+id: donations_manager
+label: 'Donations manager'
+weight: 5
+is_admin: null
+permissions:
+  - 'access toolbar'
```
Comment (kalabro) [inline modules/features/falcon_feature_user/config/install/user.role.donations_manager.yml]: Correct me if I'm wrong but I'd give this user the permissions to view orders (at least view).

File: modules/features/falcon_feature_user/config/install/user.role.manager.yml [hunk 9]
```diff
@@ -0,0 +1,13 @@
...
+id: manager
+label: Manager
+weight: 2
+is_admin: null
+permissions:
+  - 'access administration pages'
+  - 'access toolbar'
+  - 'schedule publishing of nodes'
+  - 'view scheduled content'
+  - 'view the administration theme'
```
Comment (kalabro) [inline modules/features/falcon_feature_user/config/install/user.role.manager.yml]: Could you please give managers permissions to manage all content and users?

File: modules/custom/falcon_development/config/install/features.bundle.falcon_feature.yml (hunk 2 above)
Comment (duozersk) [reply modules/custom/falcon_development/config/install/features.bundle.falcon_feature.yml:14]: By default features do not export user roles permissions (cleans them up from user role configs)... setting it to `false` allows to export permissions with roles.

File: modules/custom/falcon_commerce/falcon_commerce.install (hunk 3 above)
Comment (duozersk) [reply modules/custom/falcon_commerce/falcon_commerce.install]: When using the `drush config-import` command the module installation takes place before the configuration import... so new configs are not there yet when this install hook fires.

File: modules/custom/falcon_commerce/falcon_commerce.install (hunk 4 above)
Comment (duozersk) [reply modules/custom/falcon_commerce/falcon_commerce.install]: I have removed this hook_install as it is not working correctly with `drush config-import`. Still a thing to think about - we might need to introduce another environment variable.

File: modules/features/falcon_feature_commerce/falcon_feature_commerce.info.yml (hunk 5 above)
Comment (duozersk) [reply modules/features/falcon_feature_commerce/falcon_feature_commerce.info.yml]: Removed.

File: docs/information_architecture.rst [hunk 10]
```diff
@@ -115,17 +115,15 @@ default commerce entities generated by this module get deleted upon installation
 
 :Order Item type:
 
-    There are 2 types of order item types depending on the submitted donation type:
-
-    - **Single Donation** [single_donation] - Defines Order Item type for single donation.
-    - **Recurring Donation** [recurring_donation] - Defines Order Item type for recurring donation.
+    There is single order item type **Donation** [donation].
```
Comment (duozersk) [inline docs/information_architecture.rst:121]: Had to change it here - we can't have different order item types added to the same order type.

File: modules/features/falcon_feature_user/config/install/user.role.anonymous.yml (hunk 6 above)
Comment (duozersk) [reply modules/features/falcon_feature_user/config/install/user.role.anonymous.yml]: Not sure about it; removed for now.

File: modules/features/falcon_feature_user/config/install/user.role.authenticated.yml (hunk 7 above)
Comment (duozersk) [reply modules/features/falcon_feature_user/config/install/user.role.authenticated.yml:8]: Removed

File: modules/features/falcon_feature_user/config/install/user.role.donations_manager.yml (hunk 8 above)
Comment (duozersk) [reply modules/features/falcon_feature_user/config/install/user.role.donations_manager.yml]: Granted permissions.

File: modules/features/falcon_feature_user/config/install/user.role.manager.yml (hunk 9 above)
Comment (duozersk) [reply modules/features/falcon_feature_user/config/install/user.role.manager.yml]: Granted.

**Review by kalabro — APPROVED** [review]

**Pull request merged**